front/src/applications/operationalStudies/views/ManageTrainSchedule.tsx

```diff
@@ -8,36 +8,34 @@ import Itinerary from 'applications/operationalStudies/components/ManageTrainSch
 import Map from 'applications/operationalStudies/components/ManageTrainSchedule/Map';
 import SpeedLimitByTagSelector from 'common/SpeedLimitByTagSelector/SpeedLimitByTagSelector';
 import PowerRestrictionSelector from 'applications/operationalStudies/components/ManageTrainSchedule/PowerRestrictionSelector';
-import submitConfAddTrainSchedules from 'applications/operationalStudies/components/ManageTrainSchedule/helpers/submitConfAddTrainSchedules';
 import adjustConfWithTrainToModify from 'applications/operationalStudies/components/ManageTrainSchedule/helpers/adjustConfWithTrainToModify';
-import { FaPen, FaPlus } from 'react-icons/fa';
-import DotsLoader from 'common/DotsLoader/DotsLoader';
 import ElectricalProfiles from 'applications/operationalStudies/components/ManageTrainSchedule/ElectricalProfiles';
 import { TrainSchedule, osrdMiddlewareApi } from 'common/api/osrdMiddlewareApi';
 import {
   getPathfindingID,
+  getRollingStockID,
   getShouldRunPathfinding,
   getTrainScheduleIDsToModify,
 } from 'reducers/osrdconf/selectors';
 import { updatePathWithCatenaries, updateShouldRunPathfinding } from 'reducers/osrdconf';
 import RollingStockSelector from 'common/RollingStockSelector/WithRollingStockSelector';
 import { osrdEditoastApi, CatenaryRange } from 'common/api/osrdEditoastApi';
-import submitConfUpdateTrainSchedules from '../components/ManageTrainSchedule/helpers/submitConfUpdateTrainSchedules';
+import Tabs from 'common/Tabs';
+import rollingStockPic from 'assets/pictures/components/train.svg';
+import pahtFindingPic from 'assets/pictures/components/pathfinding.svg';
+import simulationSettings from 'assets/pictures/components/simulationSettings.svg';
+import addingSettingsPic from 'assets/pictures/components/addingSettings.svg';
 
-type Props = {
-  setDisplayTrainScheduleManagement: (arg0: string) => void;
-};
-
-export default function ManageTrainSchedule({ setDisplayTrainScheduleManagement }: Props) {
+export default function ManageTrainSchedule() {
   const dispatch = useDispatch();
   const shouldRunPathfinding = useSelector(getShouldRunPathfinding);
   const [mustUpdatePathfinding, setMustUpdatePathfinding] = useState<boolean | undefined>(
     undefined
   );
   const { t } = useTranslation(['operationalStudies/manageTrainSchedule']);
-  const [isWorking, setIsWorking] = useState(false);
+  const rollingStockID = useSelector(getRollingStockID);
   const pathFindingID = useSelector(getPathfindingID);
-  const trainScheduleIDsToModify = useSelector(getTrainScheduleIDsToModify);
+  const trainScheduleIDsToModify: undefined | number[] = useSelector(getTrainScheduleIDsToModify);
```

Comment:
```suggestion
  const trainScheduleIDsToModify: number[] = useSelector(getTrainScheduleIDsToModify | []);
```

Reply:
Impossible, syntax is wrong. Moreover, trainScheduleIDsToModify is undefined when empty